parameter_maps/IsmrmrdParameterMap.xsl

```diff
@@ -22,8 +22,9 @@ xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
 <xsl:variable name="partialFourierPhase">
   <xsl:choose>
 	<xsl:when test="siemens/MEAS/sKSpace/ucPhasePartialFourier = 1">0.5</xsl:when>
-	<xsl:when test="siemens/MEAS/sKSpace/ucPhasePartialFourier = 2">0.75</xsl:when>
-	<xsl:when test="siemens/MEAS/sKSpace/ucPhasePartialFourier = 2">0.875</xsl:when>
```

Comment:
I think that this line is almost definitely a typo, as the value of 2 is duplicated.